com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -30,7 +30,7 @@ class TempPassData {};
             {
                 scriptableRenderContext = renderContext,
                 commandBuffer = commandBuffer,
-                currentFrameIndex = GetFrameCount()
+                currentFrameIndex = m_FrameCount
```

Comment:
Is intended to still use m_FrameCount here and not the one from the camera?

Reply:
it's intented, i investigated a bit how the frame count was used by the rendergraph, and it seems it should not be camera dependent, but only you can confirm 😄

Reply:
I will take another look and try to confirm then.

Reply:
I think the change is ok yes :)